src/main/java/org/folio/verticle/DataImportConsumerVerticle.java

```diff
@@ -83,6 +84,7 @@ public void start(Promise<Void> startPromise) {
         .globalLoadSensor(globalLoadSensor)
         .subscriptionDefinition(subscriptionDefinition)
         .processRecordErrorHandler(getErrorHandler())
+        .groupInstanceId(getClass().getSimpleName() + "-" + UUID.randomUUID())
```

Comment:
It can help to resolve consumer rebalancing issue that appears in logs?

Reply:
this change sets "group.instance.id" parameter for underlying consumer that may prevent the occurence of MemberIdRequiredException. Also, it can reduce time of rebalancing process because Kafka will retain previous consumer assingment